doc/index.rst

```diff
@@ -24,8 +24,10 @@ This documents extras plugins of DNF:
 .. toctree::
    :maxdepth: 1
 
+   kickstart
```

Comment:
Eh, why is `kickstart` added here?

Reply:
Whoops, that slipped in....

I had a separate branch I was using to play with the doc stuff and noticed that kickstart wasn't listed in the documentation TOC.

Should kickstart be omitted from the doc TOC?

Reply:
No, it should be there, just in added in a different PR. :)